src/Form/CaptchaPointDeleteForm.php

```diff
@@ -41,7 +41,7 @@ public function getConfirmText() {
    */
   public function submitForm(array &$form, FormStateInterface $form_state) {
     $this->entity->delete();
-    drupal_set_message($this->t('Captcha point %label has been deleted.', array('%label' => $this->entity->label())));
+    drupal_set_message($this->t('Captcha point %label has been deleted.', array('%label' => $this->entity->getOriginalId())));
     $form_state->setRedirectUrl($this->getCancelUrl());
```

Comment:
Define "didn't work"? There is a label. Maybe it's not what was expected, but then we should update the tests instead...